lib/rbeapi/api/users.rb

```diff
@@ -51,7 +51,7 @@ def initialize(node)
         @users_re = Regexp.new(/^username\s+(?<user>[^\s]+)\s+
                                 privilege\s+(?<priv>\d+)
                                 (\s+role\s+(?<role>\S+))?
-                                (?:\s+(?<nopasswd>(nopassword)))?
+                                (?:\s+(?<nopassword>(nopassword)))?
```

Comment:
This change is not required because the key is used only internally within the module. Having said that, it is a good idea to be consistent on naming keys in hashes. <Note to self>